api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -266,7 +264,7 @@ public void setUp() throws IOException {
 
   @Test(expected = BadRequestException.class)
   public void testCreateAccount_invitationKeyMismatch() {
-    createUser();
+    createUserWithVerifiedAffiliation();
```

Comment:
Affiliation is now required for user creation.  Most tests want this, but a few use an alternative version.